src/tests/multihost/ad/test_adparameters_ported.py

```diff
@@ -3535,3 +3538,87 @@ def test_0042_ad_parameters_nonroot_user_sssd(
+            multihost, adjoin, create_aduser_group):
+        """
+        :title: IDM-SSSD-TC: SSSD does not use kerberos port that is set.
+        :bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1859315
+        :id: 558f692b-01c5-46f4-ad39-6b190dd7c017
+        :steps:
+          1. Start SSSD with alternate port in config
+          2. Call 'kinit username@domain'
+          3. Call 'ssh -l username@domain localhost' and check sssd logs
+        :expectedresults:
+          1. SSSD should start
+          2. Should succeed
+          3. Logs contain info about right port being used
+             Logs do not contain wrong (default) port being used
+        """
+
+        adjoin(membersw='adcli')
+        ad_realm = multihost.ad[0].domainname.upper()
+
+        # Create AD user and group
+        (aduser, _) = create_aduser_group
+
+        # Configure sssd
+        client = sssdTools(multihost.client[0], multihost.ad[0])
+        client.backup_sssd_conf()
+
+        dom_section = f'domain/{client.get_domain_section_name()}'
+        sssd_params = {
+            'ad_domain': multihost.ad[0].domainname,
+            'debug_level': '0x4000',
+            'cache_credentials': 'True',
+            'krb5_store_password_if_offline': 'True',
+            'krb5_server': f'{multihost.ad[0].hostname}:6666',
+            'id_provider': 'ad',
+            'auth_provider': 'krb5',
+            'access_provider': 'ad',
+            'krb5_realm': ad_realm,
+            'fallback_homedir': '/home/%u',
+        }
+        client.sssd_conf(dom_section, sssd_params)
+
+        # Clear cache and restart SSSD
+        client.clear_sssd_cache()
+
+        # Debug
+        multihost.client[0].run_command(
+            'cat /etc/sssd/sssd.conf', raiseonerr=False)
+
+        # Run kinit for the user
+        kinit_cmd = multihost.client[0].run_command(
+            f'kinit {aduser}@{ad_realm}', stdin_text='Secret123',
+            raiseonerr=False)
+
+        # Run ssh
+        multihost.client[0].run_command(
+            f'ssh -l {aduser}@{ad_realm} localhost whoami',
+            stdin_text='Secret123',
+            raiseonerr=False)
+
+        multihost.client[0].run_command(
+            'cat  /var/lib/sss/pubconf/kdcinfo.*', raiseonerr=False)
+
+        # Download all logs
+        log_str = multihost.client[0].run_command(
+            "cat /var/log/sssd/*.log").stdout_text
+
+        # TEARDOWN
+        client.restore_sssd_conf()
+        client.clear_sssd_cache()
+
+        # Evaluate test results
+        assert f"Option krb5_server has value " \
+               f"{multihost.ad[0].sys_hostname}:6666" in log_str
+        assert f"Sending initial UDP request to dgram " \
+               f"{multihost.ad[0].ip}:88" not in log_str
+        assert f"Initiating TCP connection to stream {multihost.ad[0].ip}:88" \
+               not in log_str
+        assert f"Initiating TCP connection to stream " \
+               f"{multihost.ad[0].ip}:6666" in log_str
```

Comment:
There seems to be some delay in the logs printing and from time to time I've seen a log line with the port 88 with an old timestamp in it. So, I don't know it will be possible to assert that the port 88 doesn't appear in the logs. As a first step, can you please invert the order of these two asserts? This way we make sure that the port 6666 is contacted and we will see if the other assert can be kept.

Reply:
The test is primarily about it not using 88, but I can switch them around and add a bit of delay to see what will happen.

Reply:
That sounds good to me. Let's see if that was something only happening on my environment or always.

Reply:
Done